src/renderers/shared/stack/reconciler/__tests__/ReactErrorBoundaries-test.js

```diff
@@ -1047,8 +1047,10 @@ describe('ReactErrorBoundaries', () => {
       'Normal2 render',
       // BrokenConstructor will abort rendering:
       'BrokenConstructor constructor [!]',
-      'ErrorBoundary unstable_handleError',
       ...(ReactDOMFeatureFlags.useFiber ? [
+        // Fiber unmounts an error boundary's children
+        'Normal componentWillUnmount',
```

Comment:
There is another `Normal componentWillUnmount` below right before the array ends.
Does this means we now call it twice? Doesn't seem right.